lib/spack/spack/build_environment.py

```diff
@@ -109,7 +109,7 @@ def __call__(self, *args, **kwargs):
         disable = env_flag(SPACK_NO_PARALLEL_MAKE)
         parallel = not disable and kwargs.get('parallel', self.jobs > 1)
 
-        if parallel:
+        if parallel and 'bazel' not in self.command:
```

Comment:
@muffgaga @adamjstewart Following from the discussion below : I'll try to move build-systems out of `build_environment` in a separate PR. Anyhow, I still think that `MakeExecutable` class is inappropriate for `bazel`. We should either have a new class like `BazelExecutable` or use plain `Executable`. Are you both fine with changing this ?